src/api/serializers.py

```diff
@@ -121,9 +121,10 @@ class Meta:
 
     def create(self, validated_data):
         role = validated_data.get("role")
+        disallowed_roles = ['reader', 'journal-manager']
 
-        if role.slug == 'reader':
-            raise serializers.ValidationError({"role": "You cannot add a user as a reader via the API."})
+        if role.slug in disallowed_roles:
+            raise serializers.ValidationError({"role": "You cannot add a user as a " + role.slug + " via the API."})
```

Comment:
I'd feel more comfortable if this logic took place inside a validator except of happening ad-hoc at `create` time:
https://www.django-rest-framework.org/api-guide/validators/